### PR TITLE
Added condition for creating id_set

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: demisto_sdk/commands/generate_docs/generate_script_doc.py [hunk 1]
```diff
@@ -43,8 +45,12 @@ def generate_script_doc(input_path, examples, output: str = None, permissions: s
...
 
         # get the script usages by the id set
-        id_set_creator = IDSetCreator(output='', print_logs=False)
-        id_set = id_set_creator.create_id_set()
+        if not os.path.isfile(DEFAULT_ID_SET_PATH):
+            id_set_creator = IDSetCreator(output='', print_logs=False)
+            id_set = id_set_creator.create_id_set()
+        else:
+            id_set = open_id_set_file(DEFAULT_ID_SET_PATH)
+
```
Comment (bakatzir) [inline demisto_sdk/commands/generate_docs/generate_script_doc.py:53]: It takes some time to create the id set, so maybe it is better to skip the `used_in` section in the script readme if non existent.
@roysagi WDYT?

Comment (roysagi) [reply demisto_sdk/commands/generate_docs/generate_script_doc.py:53]: I think the best course of action here is to make it interactable - ask the user if he would like us to create the id_set - if no just skip it and print a warning about what would be missing.

Comment (Shellyber) [reply demisto_sdk/commands/generate_docs/generate_script_doc.py:53]: @roysagi I can't, I'm doing this cause it takes time to create the id_set and I want to reduce it for the simple contribution flow. This runs automatically while generating the script doc

**Comment by Shellyber** [comment]
At the moment we do not support empty id_set.. 
Need to think if we want to support this.
@roysagi

**Comment by roysagi** [comment]
We should support generating docs without an available id set - we should open an issue about that.
That means the current PR's feature is good as is without any need for additional changes - @bakatzir FYI

**Review by roysagi — APPROVED** [review]
Please add a changelog

**Pull request merged**